packages/pigment-css-react/src/index.ts

```diff
@@ -6,5 +6,4 @@ export { generateAtomics, atomics } from './generateAtomics';
 export { default as css } from './css';
 export { default as createUseThemeProps } from './createUseThemeProps';
 export { default as internal_createExtendSxProp } from './createExtendSxProp';
-export { default as Box } from './Box';
```

Comment:
@brijeshb42 I have to remove this, otherwise the test fail due to "Cannot find a module ./Box". I believe that Box should be imported from `@pigment-css/react/Box` right?